### PR TITLE
Episode 2 Homework

### PR DESCRIPTION
**Pull request by johnnygoodman**

### CODE REVIEW

**Comment by johnnygoodman** [comment]
I'll stop here because:
1. I want to be sure I'm on the right track before heading into the extra credit. 
2. I've got to let that extra credit stuff marinate, maybe then it'll be...pliable. 
3. There's a "3D Trampoline Room" in Houston I just heard about and my wife and I are going to go play dodgeball. (http://houston.skyzonesports.com/Waiver.aspx)

File: spec/turn_spec.rb [hunk 1]
```diff
@@ -6,6 +6,9 @@
 	let(:strike_b) { Move.new(:strike) }
 
 	it "should have a winner" do
+	  puts "winner: " +  Turn.new(strike_a, strike_b).winner.inspect
```
Comment (jwo) [inline spec/turn_spec.rb:9]: I like to keep the puts clean for tests, so in this case I might:
1. Rails.logger.debug (and then check the log/test.log)
2. Remove the puts after they serve their purpose

File: superfight.rb [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
-puts "What is your second fighter's name?"
-fighter_b = $stdin.gets
-
-match = Match.new(Fighter.new(fighter_a), Fighter.new(fighter_b))
-
-puts "The winner of match is ....... #{match.winner.name}"
+fighter_a = "Nancy Botwin" 
+fighter_b = "Celia Hodes" 
+
+for i in 1..13 do
```
Comment (jwo) [inline superfight.rb:6]: With the 13 -- it's really 13 turns in a single match... If you want to do 13 matches, that's fine too.

Also: Try to not use for loops in ruby. If you want to loop over something and get the index, you can do this:

```
collection.each_with_index do |item, index|
end
```

**Comment by johnnygoodman** [comment]
Thanks for your feedback. I am not as far along as I hoped. 

I've deleted my repo, cloned a fresh copy and will be coming back with these changes in place and some extra credit questions for tonight's office hours session.

**Pull request closed without merging**